pyproject.toml

```diff
@@ -24,7 +24,8 @@ dependencies = [
     "pytz>=2023.3",
     "treelib>=1.2.5",
     "tqdm>=4.60.0",
-    "nvtx>=0.2.8"
+    "nvtx>=0.2.8",
+    "onnx>=1.14.0",
```

Comment:
Whats onnx added for here?

Reply:
Is it DGL requires it?

Reply:
No, actually we need it for the inference pytests : https://github.com/NVIDIA/modulus/blob/main/test/models/common/inference.py#L19